firmware/application/apps/ui_settings.cpp

```diff
@@ -221,10 +233,23 @@ SetRadioView::SetRadioView(
         send_system_refresh();
     };
 
+    force_tcxo.set_value(pmem::config_force_tcxo());
+    force_tcxo.on_select = [this](Checkbox&, bool v) {
+        if (v) {
```

Comment:
Can you just pass ‘v’ into choose_reference? Won’t need the if/else in that case.

Reply:
Oh true, great find. I have fixed that now